src/__experimental__/components/date/date.stories.js

```diff
@@ -31,7 +31,7 @@ const store = new Store(
 
 const setValue = (ev) => {
   action('onChange')(ev);
-  store.set({ value: ev.target.value });
+  store.set({ value: ev.target.value.displayText });
```

Comment:
Where does `displayText` come from? :thinking:

Reply:
Oh good spot I missed that when I refactored I will correct it thanks

Reply:
Thanks for fixing that in 8aa3379 :thumbsup: